spec/models/aroc_week_1_spec.rb

```diff
@@ -9,16 +9,15 @@
 # Here are the docs associated with ActiveRecord queries: http://guides.rubyonrails.org/active_record_querying.html
 
 # ----------------------
-
-
   it '1. finds orders by amount' do
     # ----------------------- Using Ruby -------------------------
-    orders_of_500 = Order.all.select { |order| order.amount == 500 }
-    orders_of_200 = Order.all.select { |order| order.amount == 200 }
+    # orders_of_500 = Order.all.select { |order| order.amount == 500 }
+    # orders_of_200 = Order.all.select { |order| order.amount == 200 }
     # ------------------------------------------------------------
 
     # ------------------ Using ActiveRecord ----------------------
-    # Solution goes here
+    orders_of_500 = Order.where("amount = 500")
+    orders_of_200 = Order.where("amount = 200")
```

Comment:
Nice work! You can also do ` Order.where("amount = ?", 500)`. This is an especially good practice when you have user input to prevent against sql injection attacks. ` Order.where("amount = ?", some_user_input)`